app/views/talks/_talk.html.erb

```diff
@@ -1,4 +1,4 @@
-<%# locals: (talk:) -%>
+<%# locals: (talk:, watched:, signed_in:) -%>
```

Comment:
@adrienpoly I believe the watched boolean alone is sufficient for the cache key, as a talk cannot be marked as watched if the user is not signed in. Including signed_in seems redundant in this case.

Reply:
if the user is not signed in we currently don't display the button at all. So I still need that information unless we change that and always display the watched button. 

That would be a more significant change as for unsigned user clicking on that button should display the signin/up modale.